src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return PRICE - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return PRICE - SHORT_TERM_DISCOUNT;
+  }
+
+  return PRICE;
```

Comment:
The function `calculateRentalCost` is well implemented following the checklist. It uses constants for values that should not change, which makes the code more readable and maintainable. The use of `if` with `return` statements simplifies the flow of the function, avoiding unnecessary `else` blocks.